src/shared/abbreviators.ts

```diff
@@ -45,16 +45,44 @@ const ABBREVIATION_LENGTHS = [
   [30, 0],
 ];
 
+const ABBREVIATION_LENGTHS_SOLANA = [
+  // On solscan.io, addresses are abbreviated in the form
+  // 1234567890...1234567890
+  [10, 10],
+
+  // On raydium.io, addresses in the token selection modal are abbreviated
+  // in the form 123456...123456
+  [6, 6],
+
+  // On jup.ag, addresses in the token selection modal are abbreviated
+  // in the form 12345...12345
+  [5, 5],
+
+  // On meteora.ag, addresses in the pool creation page are abbreviated
+  // in the form 1234...1234
+  [4, 4],
+];
+
 export function abbreviatedAddresses(address: string): string[] {
```

Comment:
Probably should rename to `abbreviatedEVMAddresses`.

Reply:
Addressed in https://github.com/rolod0x/rolod0x/pull/490/commits/d5db96d1a9a550dc39d8c31a82100ca86634e708